server/router/router.go

```diff
@@ -34,6 +35,11 @@ func Load(serveHTTP func(w http.ResponseWriter, r *http.Request), middleware ...
 	e := gin.New()
 	e.Use(gin.Recovery())
 
+	e.Use(func(c *gin.Context) {
```

Comment:
This is logging all incoming requests, right?

Reply:
in trace mode you do like to see everything woodpecker is doing - normal logging level is Error/Warning/Debug; Trace is for development :)

so yes you see each incommoding request